File: app/src/main/java/com/artline/muztus/audio/MusicPlayerService.java
package com.artline.muztus.audio;

import android.content.Context;
import android.media.MediaPlayer;
import android.util.Log;
import android.widget.Toast;

import com.artline.muztus.Premia;
import com.artline.muztus.R;

public class MusicPlayerService {
    private static final String TAG = "MusicManager";
    public static final int MUSIC_MENU = 0;

    private static MediaPlayer mp;
    private static int soundPosition = 0;


    public static void start(Context context, int music) {
        startM(context, music);
    }

    private static void startM(Context context, int music) {
        mp = MediaPlayer.create(context, R.raw.background_music);
        if (!mp.isPlaying()) {
            mp.start();
        }
    }

    private static boolean isPlaying() {
        return mp != null && mp.isPlaying();
    }

    public static void setPosition(Context context){
        soundPosition = mp.getCurrentPosition();
        Toast.makeText(context, soundPosition,Toast.LENGTH_LONG).show();
    }



    public static void pause() {
        if (mp != null) {
            soundPosition = mp.getCurrentPosition();
            mp.pause();
        }
    }

    public static void resume(Context context) {
        if (mp != null) {
            System.out.println("not null" + soundPosition);

            mp.seekTo(soundPosition);
            mp.start();
        } else {
            System.out.println(" null**");

            mp = MediaPlayer.create(context, R.raw.background_music);
            mp.start();
        }
    }


    public static void release() {

        Log.d(TAG, "Releasing media players");

            try {
                if (mp != null) {
                    if (mp.isPlaying()) {
                        mp.stop();
                    }
                    mp.release();
                }
            } catch (Exception e) {
                Log.e(TAG, e.getMessage(), e);
            }
    }
}

File: sounds/src/main/java/com/artline/muztus/sounds/MusicPlayerService.kt
package com.artline.muztus.sounds

import android.content.Context
import android.media.MediaPlayer

class MusicPlayerService(context: Context) {

    private val mediaPlayer: MediaPlayer by lazy {
        MediaPlayer.create(context, BG_MUSIC_RES)
    }
    private val BG_MUSIC_RES = R.raw.background_music
    private var soundPosition = 0

    fun start() {
        if (!mediaPlayer.isPlaying) {
            mediaPlayer.start()
        }
    }

    fun pause() {
        soundPosition = mediaPlayer.currentPosition
        mediaPlayer.pause()
    }

    fun resume() {
        mediaPlayer.seekTo(soundPosition)
        mediaPlayer.start()
    }
}